### ISSUE TITLE
When user clicks "Linked Data Editor" or goes directly to sinopia.io/editor, take them to the "Resource Templates" tab

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#480]
in order to catalog, user has to first select a template, which is done on the Resource Templates tab, so take the user to this tab, not to Editor tab, as is current behavior.

see also #479

**Comment by jgreben** [comment]
Fixed by #484

**Comment by michelleif** [comment]
just realized that the "Resource Templates" tab requires you to log in before you can do anything, 
but our plan before was to allow  people to be able to go to the Browse tab and look at stuff without having to log in, and to go to the Editor tab and explore templates without having to log in.

so now that we've made "Resource Templates" be the default when someone clicks "Linked Data Editor", there is now no way to get to the Browse tab or the Editor tab without being forced to log in first.

can we revert to the Linked Data Editor taking user to Editor tab, like it was originally?

**Comment by jermnelson** [comment]
Hi @michelleif, non-authenicated users can go to the editor tab directly by typing this url https://development.sinopia.io/editor/, if they click on the `Resource Templates` tab, they will then be required to login as we would expect.

**Comment by jgreben** [comment]
@michelleif I created issue #495 to follow on PR #491. This solution (when finished) will open up the editor to anyone to choose existing resource templates (previously uploaded by _authenticated_ users), work with those RTs and preview the RDF, but will prevent them from uploading their own resource templates (or save RDF when we get to that point).

Let me know if you think this solution will work. I created a tentative storytime topic to discuss this if need be.
